dpdd.tex

```diff
@@ -656,7 +657,7 @@ \subsubsection{\SSObject Table}
     \item Removed all asteroid shape-related columns; determining these is outside of the scope of the Project.
     \item Removed taxonomy related columns; determining these is outside of the scope of the Project.
     \item Removed \texttt{albedo} -- I don't believe albedo can be determined solely from LSST data. More likely, we will need to assume a particular value. If this value is not universal, this column will need to be put back in.
-    \item Removed \texttt{xMag, xMagErr, xAmplitude, xPeriod} columns as they were not clearly defined (eg., w/o phase correction, do they make sense?). These can be recovered by querying the \DIASource table for magnitudes. Deriving phase-corrected light curves is left as a Level 3 task.
+    \item Removed \texttt{xMag, xMagErr, xAmplitude, xPeriod} columns as they were not clearly defined (eg., w/o phase correction, do they make sense?). These can be recovered by querying the \DIASource table for magnitudes. Deriving phase-corrected light curves is left as a User Generated task.
```

Comment:
I'm not sure I like "User Generated task" here. By "User Generated" don't we really mean the products generated by science users? Here we are talking about an activity that a science user might want to do. Would it be clearer to say "Deriving phase-corrected light curves will be possible in the LSST Science Platform environment."?

Reply:
Yes, "Level 3" has had two meanings: one for the data products, which are now indeed "User Generated" but the other for the processing and services used to generate those data products, which could now be labeled "science user" or "LSST Science Platform".

Reply:
Note that there are now several other instances of "User Generated" task that need to be rethought; I'm not going to call each out.  (Also note that "eg." should be "e.g.".)

Reply:
I have put " Deriving phase-corrected light curves should now be done in the Science Platform.
"
I fixed a couple of other sentences like this also